compiler-cli/src/dependencies.rs

```diff
@@ -1075,8 +1078,8 @@ fn provide_package(
     match provided.get(&package_name) {
         Some(package) if package.source == package_source => {
             // This package has already been provided from this source, return the version
-            let version = hexpm::version::Range::new(format!("== {}", &package.version));
-            return Ok(version);
+            let version = GleamVersion::new(format!("== {}", &package.version))?;
+            return Ok(version.hex().clone());
```

Comment:
Could make a function here that consumes the version rather than taking a reference, cloning, and throwing away the original.

```suggestion
            return Ok(version.into_hex());
```

Can be used on line 1139 also

Reply:
Fixed :D